docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -147,6 +152,15 @@ Type | Key | Value | Default value | Description
 &#xfeff;                                                     | weight_regularizer_constant | (float)                     | 1                       | Weight regularizer constant
 `ML_TRAIN_LAYER_TYPE_FC`                                     |                             |                             |                         | Fully connected layer
 &#xfeff;                                                     | unit                        | (unsigned integer)          |                         | Number of outputs
+`ML_TRAIN_LAYER_TYPE_CONV1D` (since 7.0)                     |                             |                             |                         | 1D Convolution layer
+&#xfeff;                                                     | filters                     | (unsigned integer)          |                         | Number of filters
+&#xfeff;                                                     | kernel_size                 | (unsigned integer)          |                         | Comma-seperated unsigned integers for kernel size, `height, width`  respectively
```

Comment:
should remove `height, width`. It is single unsigned integer.

Reply:
thank you